.github/workflows/nsis.yml

```diff
@@ -0,0 +1,68 @@
+name: nsis
+
+on:
+  workflow_run:
+    workflows: [ Build ]
+    types: [ completed ]
+  workflow_dispatch:
+
+jobs:
+  config:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.files.outputs.matrix }}
+      branch: ${{ steps.branch-ref.outputs.name }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.event.workflow_run.head_branch }}
+
+      - name: Return Branch
+        id: branch-ref
+        run: echo "::set-output name=${{ github.event.workflow_run.head_branch }}"
+
+      - name: Get Script Files
+        id: list-files
+        uses: Rishabh510/Path-lister-action@master
+        with:
+          path: installers/
+          type: .nsi
+
+      - name: Return Script Files
+        id: files
+        run: echo "::set-output name=matrix::$(echo "${{ steps.list-files.outputs.paths }}" | sed 's/\//\\/g' | jq --raw-input --compact-output '[splits(" ") | select(length > 0)]')"
+
+  make-installer:
+    runs-on: windows-latest
+    needs: [ config ]
+    strategy:
+      fail-fast: false
+      matrix:
+        file: ${{ fromJson(needs.config.outputs.matrix) }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ github.event.workflow_run.head_branch }}
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+          workflow_conclusion: success
+          name: hobovr-build-windows-latest
+          path: ${{ github.workspace }}
+
+      - name: Create nsis installer
+        id: create-installer
+        shell: powershell
+        run: |
+          "$CONDA\bin\conda.bat execute ${{ matrix.file }}"
+        working-directory: ${{ github.workspace }}\installers\win
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: "hobovr_setup.exe" # Get the Output Name
```

Comment:
This really needs to be a value extracted from the .nsi file in the future, especially if we have multiple outputs.
I'll add this to the TODO of #5

Reply:
Honestly, this is a pretty big change already that's been in the works for about a week, so I think moving extra work into todos for after merging is a good idea. I'm not really sure how hard extracting the exe name from the nsi file and putting it in the yml will be though.

Reply:
@SimLeek I've got this issue about 90% solved using SED in a separate branch already.

Should I fold it into this PR?